ui/app/src/modules/Search/FilterSearchDropdown.tsx

```diff
@@ -540,14 +548,32 @@ function SortOrder(props: SortOrderProps) {
   const classes = useStyles({});
   const { formatMessage } = useIntl();
   const { queryParams, handleFilterChange } = props;
+  // queryParams['sortBy'] === undefined: this means the current filter is the default === _score
```

Comment:
Sure? There's no value on the select for relevance? 🤔

Reply:
that means when there are not a value on the URL, because Im not sending the default values at the beginning